dev/core#5845 setup: move Sample Data lower, tweak the wording

mlutfy opened the pull request:
Overview
----------------------------------------

In the web installer, move the part about "Sample Data" towards the bottom of the install page, and clarify the wording a bit, normal font size.

Also removes the redundant "Thank you for choosing CiviCRM" and "CiviCRM Installer" title.

Related to: https://lab.civicrm.org/dev/core/-/issues/5845 and #32610

Before
----------------------------------------

![image](https://github.com/user-attachments/assets/2ed73c56-6064-4a5e-85d7-80c7de39b378)

After
----------------------------------------

![image](https://github.com/user-attachments/assets/7ca4946a-fbcb-4ab6-96ba-18d57e3484a7)

Comments
------------------------------

The slide effect on the top h1 was entertaining, and I tried to keep it on the first paragraph, but it felt off.

mlutfy committed:
setup: move Sample Data lower, tweak the wording, remove redundant thanks

eileenmcnaughton commented:
@vingle any opinion?

totten commented:
* I like how it moves "Sample Data". :+1: 
* I think it's good for any page to show a title (like "__CiviCRM Installer__"). So you know what page you're on. :compass: :world_map:
* "Thank you" seems well and good, but... yeah... the duplicate "Thank you"s are little much. (*I have no opinion on which variant to keep/move/remove/edit.*)

mlutfy commented:
I did a few tweaks:

- Sample data becomes Test data
- Added back the main "CiviCRM Installer" title
- Added back the slide effect, but on the first sentence
- Added a rocket, because it goes well with the slide effect, but perhaps 🌱 or 🧭 would be less environmentally destructive / tech-bro  (I agree, software should be fun, even if it's an installer; even my boring banking app has party-poppers when I do transactions)

[Screencast from 2025-04-18 11-36-44.webm](https://github.com/user-attachments/assets/dc07798c-812c-431e-9db6-ee50c2807804)

kcristiano commented:
Thanks @mlutfy I like this.  My objection is not for myself as There are other ways to add the data.  I find it helps people exploring CiviCRM on their own (the DIY crowd) as we do get folks who have installed CiviCRM and used the sample date to test things out themselves.   I think having it as an option is needed and this does it well.

mlutfy commented:
Thanks for the feedback @kcristiano. This PR keeps the sample data, and I don't plan on removing it for now.

This PR also keeps the welcome slide/animation.

I think all the other points raised were addressed.

ufundo commented:
Personally quibbles:
- the fly-in feels a little bit early 00s PowerPoint to me
- I would def go for growing leaf emoji over rocket *
- I would emoji at the end of sentence rather than the beginning

But I agree all the points have been addressed and there is lots of big improvements here.

* a passage I come back to a lot 
![image](https://github.com/user-attachments/assets/54e65756-c566-4bdd-8fe7-8d75192222b3)
from https://aspirationtech.org/publications/manifesto

mlutfy commented:
Seedling it is :)

![image](https://github.com/user-attachments/assets/1ef955f0-c2bd-41d0-9834-7c359006eb72)

(I think a cactus would have been more representative of CiviCRM) :joy: 

but I left the slide in, because no strong opinions.
